File: core/src/main/java/com/minirogue/starwarscanontracker/core/model/room/entity/MediaTypeDto.java
package com.minirogue.starwarscanontracker.core.model.room.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "media_types")
public class MediaTypeDto {
    @PrimaryKey
    @ColumnInfo
    private int id;
    @ColumnInfo
    private String text;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: core/src/main/java/com/minirogue/starwarscanontracker/core/model/room/entity/MediaTypeDto.kt
package com.minirogue.starwarscanontracker.core.model.room.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "media_types")
data class MediaTypeDto(
    @PrimaryKey
    @ColumnInfo
    val id: Int,
    @ColumnInfo
    val text: String,
)
